integration_tests/models/generic_tests/test_recency.sql

```diff
@@ -1,16 +0,0 @@
-
-{% if target.type == 'postgres' %}
-
-select
-    1 as col1,
-    2 as col2,
-    {{ dbt_utils.date_trunc('day', dbt_utils.current_timestamp()) }} as today
```

Comment:
Are you deleting this file because it's using timestamp? We should point to the dbt Core provided backcompat macro instead I think

Reply:
I think this got deleted during merge somehow